nistats/design_matrix.py

```diff
@@ -61,38 +69,39 @@ def _poly_drift(order, frame_times):
 
 
 def _cosine_drift(period_cut, frame_times):
-    """Create a cosine drift matrix with periods greater or equals to
+    """Create a cosine drift matrix with periods greater or equal to
     period_cut.
 
     Parameters
     ----------
-    period_cut: float 
+    period_cut : float
          Cut period of the low-pass filter (in sec)
-    frame_times: array of shape(n_scans)
+
+    frame_times : array of shape(n_scans)
          The sampling times (in sec)
 
     Returns
     -------
-    cdrift:  array of shape(n_scans, n_drifts)
-             cosin drifts plus a constant regressor at cdrift[:,0]
+    cosine_drift:  array of shape(n_scans, n_drifts)
```

Comment:
You probably want to add a space before the colon while you are fixing up this docstring.